protocol/synthetix/test/integration/modules/core/CollateralModule/CollateralModule.deposit.test.ts

```diff
@@ -147,6 +148,32 @@ describe('CollateralModule', function () {
               });
             });
 
+            describe('when attempting to withdraw locked collateral', () => {
+              var lockedTill;
+              it('get current time', async () => {
+                const currentTime = await getTime(provider());
+                lockedTill = currentTime + 60 * 60 * 24 * 30;
+              });
+              it('reverts', async () => {
+                await systems()
```

Comment:
in accordance with our testing patterns, please move this call into a separate `before` section